cmd/nrt/config_split/config.toml

```diff
@@ -5,11 +5,13 @@ TrimColAfterSplit = true # (bool), if true, designated Trim Columns apply to Spl
 WalkSubFolders = true # (bool), if true, splitter process every file including the file in sub-folders; otherwise, ignores sub-folder files.
 
 [Trim]
-Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.      
+Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.                    
 Enabled = true # (bool), turn on/off Trim function.
-OutFolder = "./out/" # (string), in which folder trimmed csv files should be output.
+OutFolder = "./out_trim/" # (string), in which folder trimmed csv files should be output.
 
 [Split]
 Enabled = true # (bool), turn on/off Split function.
```

Comment:
Request we leave this disabled by default, it does add to execution time (and from what I'm seeing, that can be quite significant when we're dealing with large RRD files). I will let TAAs know of the availability of the functionality as we release the tool.